src/Compiler/CompilerOptions.cpp

```diff
@@ -281,6 +282,13 @@ static llvm::cl::opt<bool, true> printIROpt("printIR",
     llvm::cl::desc("Print the IR to stdout:."), llvm::cl::location(printIR),
     llvm::cl::init(false), llvm::cl::cat(OnnxMlirOptions));
 
+static llvm::cl::opt<bool, true> emitElidedIROnlyOpt("emitElidedIROnly",
```

Comment:
`emitElidedIROnly` is a bit confusing in the sense that it looks like other `EmitONNXIR`, `EmitMLIR`, ... How about `doNotEmitFullMLIRCode` (`--do-not-emit-full-mlir-code`)?

Reply:
Updated the option name. Thanks!

Reply:
Also, added the lit test.